tests/10-issue.js

```diff
@@ -26,95 +26,284 @@ describe('BitstringStatusList Credentials (Issue "statusPurpose: revocation")',
+                  statusReferenceType.should.be.oneOf(['string', 'array'],
+                    'Expected statusReference to be an string or an array.');
+                  // TODO test for URLS
+                });
+              } else {
+                this.skip();
+              }
+            });
         });
-        it('MUST be 16kb in size when "encodedList" is decoded',
+        describe('BitstringStatusList Credential', function() {
+          let statusListCredential;
+          before(async function() {
+            statusListCredential = (await getSlc({issuedVc})).slc;
+          });
+          it('When a status list verifiable credential is published, ' +
+            'it MUST be a conforming document, ' +
+            'as defined in [VC-DATA-MODEL-2.0].',
+          async function() {
+            testSlCredential({slCredential: statusListCredential},
+              'Expected status credential to conform to VCDM 2.0.');
+          });
+          it('The verifiable credential that contains the status list ' +
+            'MUST express a type property that includes the ' +
+            'BitstringStatusListCredential value.',
+          async function() {
+            statusListCredential.hasOwnProperty('type',
+              'Expected credential status to have a type.');
+            statusListCredential.type.should.include.members([
+              'BitstringStatusListCredential'
+            ],
+            'Expected credential status type to include ' +
+            'BitstringStatusListCredential.');
+          }
+          );
+          it('The type of the credential subject, which is the status list, ' +
+            'MUST be BitstringStatusList.',
+          async function() {
+            statusListCredential.credentialSubject.hasOwnProperty('type',
+              'Expected credentialSubject to have a type.');
+            const credentialSubjectType = typeof (
+              statusListCredential.credentialSubject.type);
+            credentialSubjectType.should.be.oneOf(['string', 'array'],
+              'Expected credential status type to be a string or an array.');
+            if(credentialSubjectType === 'string') {
+              statusListCredential.credentialSubject.type = [
+                statusListCredential.credentialSubject.type];
+            }
+            statusListCredential.credentialSubject.type.should.include.members([
+              'BitstringStatusList'
+            ],
+            'Expected credential status type to include BitstringStatusList.');
+          });
+          it('The value of the purpose property of the status entry, ' +
+            'statusPurpose, MUST be one or more strings.',
+          async function() {
+            const statusPurposeType = typeof (
+              statusListCredential.credentialSubject.statusPurpose);
+            statusPurposeType.should.be.oneOf(['string', 'object']);
+            if(statusPurposeType === ('object')) {
```

Comment:
```suggestion
            if(statusPurposeType === 'object') {
```

Remove redundant parenthesis. Save those for Lisp coding. 😁